shared/templates/sudo_defaults_option/ansible.template

```diff
@@ -3,10 +3,28 @@
 # strategy = restrict
 # complexity = low
 # disruption = low
+{{% if VARIABLE_NAME %}}
+{{{ ansible_instantiate_variables(VARIABLE_NAME) }}}
+- name: Ensure {{{ OPTION }}} is enabled with the appropriate value in /etc/sudoers
+  lineinfile:
+    path: /etc/sudoers
+    regexp: '^[\s]*Defaults\s(.*)\b{{{ OPTION }}}=\w+\b(.*)$'
+    line: 'Defaults \g<1>{{{ OPTION }}}={{{ "{{ " ~ VARIABLE_NAME ~ " }}" }}}\g<2>'
```

Comment:
Is there a reason why you choose `\g<1>` over just `\1`?
And why not to write just `{{{ OPTION }}}={{ {{{ VARIABLE_NAME }}} }}`?

Reply:
Very smart you are. Updated per suggestion.
The named backreferences were reminiscing from other Ansible snippets I took as inspiration. They are not required for it to work.